src/studio-header/BrandNav.jsx

```diff
@@ -1,24 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Hyperlink } from '@openedx/paragon';
+
+import { navigateToUrl } from './utils';
 
 const BrandNav = ({
   studioBaseUrl,
   logo,
   logoAltText,
+  onNavigate,
 }) => (
-  <a href={studioBaseUrl}>
+  <Hyperlink destination={studioBaseUrl} onClick={(e) => navigateToUrl(e, studioBaseUrl, onNavigate)}>
```

Comment:
This is OK, but isn't it much simpler and nicer to just use the [`<Link>`](https://reactrouter.com/en/main/components/link) component from `react-router-dom`?

Reply:
> I'm missing an explanation of why we're doing this complicated onNavigate prop injection and duplicating logic from react-router instead of simply using <Link>.
The reason why it is not possible to use the react-router-dom functionality is the lack of a react-router-dom context, in [the Authoring MFE](https://github.com/openedx/frontend-app-authoring/blob/master/src/CourseAuthoringPage.jsx) the Header is outside the react-router context of the app. If you have any workarounds, I will be happy to implement them

Reply:
Are you sure? I thought [the root app always provides `RouterProvider`](https://github.com/openedx/frontend-app-authoring/blob/ecfe27b043965941d752541e7b3e512847d94169/src/index.jsx#L93-L97). And in [that page you linked to](https://github.com/openedx/frontend-app-authoring/blob/master/src/CourseAuthoringPage.jsx), I can see the use of `useLocation` from react-router which means the router context is available.